home-work-app/src/app/pages/home/components/AddMovieLink/AddMovieLink.tsx

```diff
@@ -1,44 +1,24 @@
 import { StyledLink } from './StyledLink.styled';
 import { AddMovieModal } from '../../../../components/modals/AddMovieModal/AddMovieModal';
 import { ModalPortal } from '../../../../components/modals/CommonModalTemplate/ModalPortal';
-
-type AddMovieLinkState = {
-    isModalOpen: boolean;
-};
+import { useState } from 'react';
 
 const modalRoot: HTMLElement = document.body;
 
-export class AddMovieLink extends React.PureComponent {
-    state: AddMovieLinkState;
-
-    constructor(props: {}) {
-        super(props);
-        this.state = {
-            isModalOpen: false
-        };
-    }
+export const AddMovieLink: React.FC = () => {
+    const [isModalOpen, toggle] = useState(false);
```

Comment:
`const [isModalOpen, setIsModalOpen] = useState(false);`
Always use `[propert, setProperty]` naming pattern for `useState` values.